.appveyor.yml

```diff
@@ -4,6 +4,7 @@ configuration:
   - Release
 platform:
   - x86
+  - x64
```

Comment:
The debug configuration could be removed if the automated build script takes too long when both x86 and x64 are built. It gets to be a lot of options.